src/main/kotlin/uk/gov/dluhc/printapi/scheduler/BatchPrintRequestsJob.kt

```diff
@@ -1,4 +1,4 @@
-package uk.gov.dluhc.printapi.jobs
+package uk.gov.dluhc.printapi.scheduler
```

Comment:
Are the classes in here "jobs" or "schedulers"? To me they are not "schedul**ers**" , in that they don't do the scheduling (Spring does that)

I'm happy that we refactor things like this if we feel fit, but it feels like it should at least warrant a team discussion ?

Reply:
I didn't rename in my PR too as I wanted to get a wider team view. I'm also of the opinion that they are jobs and not schedulers

Reply:
Right, I have moved it back to jobs package. (Changes pushed)